images/rabbitmq/definitions.json

```diff
@@ -0,0 +1,14 @@
+{
+  "vhosts":[
+        {"name": "/"}
```

Comment:
let's make this also into an Env Variable.

Reply:
I agree.

Reply:
Shall we create other Env Variables like the name of the policy or perhaps the ha-mode (es: 'all')?

Reply:
no, if somebody want's to change that deep I would suggest they update the whole `definitions.json` file themselves :)